### PR TITLE
STIJ-211: Fixed styling of newsletter block.

### PR DESCRIPTION
**Pull request by meirege**
## Description
Removed the borders introduced by adding a themified border to the island component.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: components/31-molecules/building-blocks/newsletter-block/_newsletter-block.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.newsletter-block {
+div.newsletter-block {
   @include desktop {
     display: flex;
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/newsletter-block/_newsletter-block.scss]: container isn't correct anymore

Comment (meirege) [reply components/31-molecules/building-blocks/newsletter-block/_newsletter-block.scss]: Should be fixed now.

File: components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig [hunk 2]
```diff
@@ -1,54 +1,54 @@
...
+        } %}
+      </div>
     </div>
-  </div>
-  <div class="island section--default newsletter-block__content">
-    <div>
+    <div class="island section--wrapper newsletter-block__social-media">
       {% include '@h3' with {
-        'h3_text': 'Subscribe to the newsletter of the City of Ghent.'
+      'h3_text': 'Volg ons op:'
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig]: English

File: components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig [hunk 3]
```diff
@@ -1,54 +1,54 @@
...
-    'items': [
-      '<a href="https://www.facebook.com/stadsbestuurgent" title="Facebook"><i class="icon-facebook"></i><span class="element-invisible">Facebook</span></a>',
-      '<a href="https://twitter.com/stadgent" title="Twitter"><i class="icon-twitter"></i><span class="element-invisible">Twitter</span></a>',
-      '<a href="https://www.linkedin.com/company/stad-gent" title="LinkedIn"><i class="icon-linkedin"></i><span class="element-invisible">LinkedIn</span></a>',
-      '<a href="https://www.youtube.com/channel/UCmJLf8SFG0D0GTd6LSSTQLw" title="Youtube"><i class="icon-youtube"></i><span class="element-invisible">Youtube</span></a>',
-      '<a href="https://www.instagram.com/stadgent/" title="Instagram"><i class="icon-instagram"></i><span class="element-invisible">Instagram</span></a>'
-      ]
-    } %}
   </div>
 </div>
```
Comment (Jeroen005) [inline components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig]: new line

File: components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig (hunk 2 above)
Comment (meirege) [reply components/31-molecules/building-blocks/newsletter-block/newsletter-block.twig]: Fixed.

**Pull request merged**